specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/networkSecurityPerimeter.json

```diff
@@ -1343,6 +1768,262 @@
+        },
+        "autoApprovedRemotePerimeterId": {
+          "type": "string",
+          "description": "Perimeter ARM Id for the remote NSP with which the link gets created in Auto-approval mode. Used when the NSP admin have access on the remote NSP resource.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "remotePerimeterGuid": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Remote NSP Guid with which the link gets created."
+        },
+        "localInboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Local Inbound profile names to which Inbound is allowed. Use ['*'] to allow inbound to all profiles. It's default value is ['*'].",
+          "uniqueItems": true
+        },
+        "localOutboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "Local Outbound profile names from which Outbound is allowed. In current version, it's value is set to ['*'] to allow outbound from all profiles. In later version, user will be able to modify it.",
+          "uniqueItems": true
+        },
+        "remoteInboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Remote Inbound profile names to which Inbound is allowed. Use ['*'] to allow inbound to all profiles. This property can only be updated in auto-approval mode. It's default value is ['*'].",
+          "uniqueItems": true
+        },
+        "remoteOutboundProfiles": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true,
+          "description": "Remote Outbound profile names from which Outbound is allowed. In current version, it's value is set to ['*'] to allow outbound from all profiles. In later version, user will be able to modify it.",
+          "uniqueItems": true
+        },
+        "description": {
+          "type": "string",
+          "description": "A message passed to the owner of the remote NSP link resource with this connection request. Restricted to 140 chars."
+        },
+        "status": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The NSP link state.",
+          "enum": [
+            "Approved",
+            "Pending",
+            "Rejected",
+            "Disconnected"
+          ],
+          "x-ms-enum": {
+            "name": "nspLinkStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Properties of NSP Link resource."
+    },
+    "NspLinkReconcile": {
+      "type": "object",
+      "properties": {},
+      "description": "Request object for NSP link reconcile."
+    },
```

Comment:
PLease define this object with full schema. Free form objects are not allowed in the swagger.

Reply:
Removed it, since we don't need any body and response.